packages/malloy/src/malloy.ts

```diff
@@ -2563,15 +2563,17 @@ class FluentState<T> {
   }
 
   protected makeQueryMaterializer(
-    materialize: () => Promise<PreparedQuery>
+    materialize: () => Promise<PreparedQuery>,
+    options?: PrepareResultOptions
   ): QueryMaterializer {
-    return new QueryMaterializer(this.runtime, materialize);
+    return new QueryMaterializer(this.runtime, materialize, options);
   }
 
   protected makeExploreMaterializer(
-    materialize: () => Promise<Explore>
+    materialize: () => Promise<Explore>,
+    options?: PrepareResultOptions
   ): ExploreMaterializer {
-    return new ExploreMaterializer(this.runtime, materialize);
+    return new ExploreMaterializer(this.runtime, materialize, options);
   }
```

Comment:
`ExploreMaterializer` can also store a `replaceMaterializedReferences` so that you can say 
```ts
await runtime
  .loadModel(..., { replaceMaterializedReferences: true })
  .loadExploreByName("foo")
  .loadQueryByName("bar")
  .run()
```

Note that `.loadExploreByName()` doesn't have the `replaceMaterializedReferences` option. I could go either way on this, but noting in loading an explore actually needs the setting...